docs/guides/operator-guide-azure-enclave.md

```diff
@@ -104,23 +98,22 @@ To deploy a new UID2 Private Operator for Azure, you'll need to complete the fol
 
 ### Download ZIP File and Extract Files
 
-The first step is to get set up with the deployment files you'll need.
+The first step is to get set up with the deployment files you'll need. Download the zip file linked in the table below for the Azure Download for the version you are installing. In most cases, you should install the latest version.
 
-Follow these steps:
-
-1. In the Azure Enclave GitHub release page that you were given after completing your UID2 account setup (see [Set Up UID2 Operator Account](#set-up-uid2-operator-account)), locate and download the ZIP file containing the files you'll need for your deployment. The ZIP file is named according to the following convention:
-
-   ```
-   uid2-operator-deployment-artifacts-{VERSION_NUMBER}-azure-cc.zip
-   ```
-
-2. Unzip the `uid2-operator-deployment-artifacts-{VERSION_NUMBER}-azure-cc.zip` file to extract the following files, needed for the deployment:
+1. Unzip the zip file to extract the following files, needed for the deployment:
 
    - `vault.json` and `vault.parameters.json`
    - `vnet.json` and `vnet.parameters.json`
    - `operator.json` and `operator.parameters.json`
    - `gateway.json` and `gateway.parameters.json`
 
+#### Operator Version
+The latest zip can be found under Azure Version Below:
```

Comment:
Seconding the comment from @thomasm-ttd re the column title. But ZIP is all caps, as an acronym, and we don't use below in the docs. Suggest:

The latest ZIP file is linked in the Azure Download column in the following table.